### DISCUSSION TITLE
Third Party Scope

### DISCUSSION THREAD

**Discussion by helenafranca** [GoogleChromeLabs/ps-analysis-tool#537] [Support Forum] [answered]
Can I assume that all cookies with a Third Party Scope will be blocked?

**Comment by amedina** [comment, marked as answer]
In general yes, but there will be cases where some 3P cookies will not be blocked. Some cases include: use of Partitioned attribute and temporary exceptions (e.g. deprecation trial, heuristics, enterprise).
